k8s/resources/common/config/loadflow-server-application.yml

```diff
@@ -1,8 +1,2 @@
 loadflow:
   default-provider: OpenLoadFlow
-
-management:
-  endpoints:
-    web:
-      exposure:
-        include: prometheus, health, info
```

Comment:
You can move it to [`k8s/resources/common/config/common-application.yml`](https://github.com/gridsuite/deployment/blob/enable-prometheus-all-ms/k8s/resources/common/config/common-application.yml) to de-duplicate configuration.

Reply:
I actually moved it even higher in the hierarchy, see https://github.com/powsybl/powsybl-ws-commons/pull/49

Reply:
oh okay, it would be good to wait that PR49 is merged+released and all services upgraded to latest powsybl-ws-common version to not loose current config, as deployment repository is automatically updated to main head.

I added a reference between the PR to keep information.